src/gen/api/accountingApi.ts

```diff
@@ -10125,72 +10192,6 @@ export class AccountingApi {
             });
         });
     }
-    /**
-     * 
-     * @summary Allows you to retrieve prepayments as PDF files
-     * @param xeroTenantId Xero identifier for Tenant
-     * @param prepaymentID Unique identifier for a PrePayment
-     */     
-    public async getPrepaymentAsPdf (xeroTenantId: string, prepaymentID: string, options: {headers: {[name: string]: string}} = {headers: {}}) : Promise<{ response: http.IncomingMessage; body: Buffer;  }> {
```

Comment:
And one to log was removed - as prepayments are not actually downloadable as pdf